src/current/_includes/releases/v23.1/v23.1.17.md

```diff
@@ -0,0 +1,103 @@
+    - `waiting_stmt_query`: the query of the statement that is waiting to run.
+    - `blocking_txn_queries_unordered`: an unordered list of the blocking transaction's queries. [#119422]
+
+    [#119422]
+
+<h3 id="v23-1-17-db-console-changes">DB Console changes</h3>
+
+- The [**Statements** page](https://www.cockroachlabs.com/docs/v23.1/ui-statements-page) now always shows the entire selected period, instead of only the period that contains data. [#118811][#118811]
+- The [**Overload Dashboard**](https://www.cockroachlabs.com/docs/v23.1/ui-overload-dashboard) page now includes two additional graphs:
+    - **Elastic CPU Utilization**: Shows the actual CPU used for elastic work compared with the configured limit.
+    - **Elastic CPU Exhausted Duration Per Second**: Shows how much time, in milliseconds, that elastic work has been subject to CPU exhaustion. [#118908][#118908]
+
+<h3 id="v23-1-17-bug-fixes">Bug fixes</h3>
+
+- Fixed a bug where creating a [changefeed](https://www.cockroachlabs.com/docs/v23.1/change-data-capture-overview) that targeted tables with a zero-scale `DECIMAL(n)` column, `format='avro'`, and `diff` would cause a panic. [#118848][#118848]
+- Fixed a bug where a [changefeed](https://www.cockroachlabs.com/docs/v23.1/change-data-capture-overview.html) could omit events in rare cases, logging the error `cdc ux violation: detected timestamp ... that is less or equal to the local frontier`. This could happen in the following scenario:
+    1. A [rangefeed](https://www.cockroachlabs.com/docs/v23.1/create-and-configure-changefeeds.html#enable-rangefeeds) runs on a follower [replica](https://www.cockroachlabs.com/docs/v23.1/architecture/glossary#cockroachdb-architecture-terms) that lags significantly behind the [leaseholder](https://www.cockroachlabs.com/docs/v23.1/architecture/glossary#cockroachdb-architecture-terms).
+    1. A transaction commits and removes its [transaction record](https://www.cockroachlabs.com/docs/v23.1/architecture/transaction-layer#transaction-records) before its [intent](https://www.cockroachlabs.com/docs/v23.1/architecture/transaction-layer#writing) resolution is applied on the follower.
+    1. The follower's [closed timestamp](https://www.cockroachlabs.com/docs/v23.1/architecture/transaction-layer#closed-timestamps) has advanced past the transaction commit timestamp.
+    1. The rangefeed attempts to push the transaction to a new timestamp (at least 10 seconds after the transaction began).
+    1. This may cause the rangefeed to prematurely emit a checkpoint before emitting writes at lower timestamps, which in turn may cause the [changefeed](https://www.cockroachlabs.com/docs/v23.1/change-data-capture-overview#how-does-an-enterprise-changefeed-work) to drop these events entirely, never emitting them.
+
+    [#118415][#118415]
+- Fixed a bug introduced in v22.2 where queries issued by the TTL jobs did not use optimal plans. [#118496][#118496]
```

Comment:
'TTL jobs' could be a link to https://www.cockroachlabs.com/docs/v23.1/row-level-ttl#view-scheduled-ttl-jobs